packages/trader/src/App/Components/Elements/PositionsDrawer/result-mobile.tsx

```diff
@@ -1,11 +1,15 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
 import { Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
 
-const ResultMobile = ({ is_visible, result }) => {
+type TResultMobile = {
+    is_visible: boolean;
+    result?: 'won' | 'lost';
```

Comment:
Why is this optional?

Reply:
because contract is still ongoing its actually undefined without result (won or lost)

Reply:
ah yup u are right. Changed.